core-libs/setup/.release-it.json

```diff
@@ -26,7 +26,7 @@
           "path": [
             "peerDependencies.@spartacus/core",
             "peerDependencies.@spartacus/storefront",
-            "peerDependencies.@spartacus/my-account"
+            "peerDependencies.@spartacus/organization"
```

Comment:
This line can be dropped. We forgot about it with lazy loading ticket. Setup lin no longer have dependency on organization